lib/faker/tv_shows/suits.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Faker
+  class TvShows
+    class Suits < Base
+      flexible :suits
+
+      class << self
+        ##
+        # Produces a character from Suits
```

Comment:
Can you add a period at the end of sentence?

```suggestion
        # Produces a character from Suits.
```